deploy/sample-backup-configs/sample_config_build_farm.yaml

```diff
@@ -0,0 +1,55 @@
+# Build-time build farm design configuration for the FireSim Simulation Manager
+# See https://docs.fires.im/en/stable/Advanced-Usage/Manager/Manager-Configuration-Files.html for documentation of all of these params.
+
+###########
+# Schema:
+###########
+# # Unique name for build farm.
+# <NAME>:
+#     # Human readable name of the build farm type.
+#     # This can be determined from the `NAME` field in `BuildFarmHostDispatcher`
+#     # child class (located in `deploy/buildfarmhostdispatcher.py`).
+#     build-farm-type: <TYPE NAME>
+#     args:
+#         # Build farm arguments that are passed to the `BuildFarmHostDispatcher`
+#         # object. Determined by looking at `parse_args` function of class.
+#         <K/V pairs of args>
+
+# Note: For large designs (ones that would fill a EC2.2xlarge/Xilinx VU9P)
+# Vivado uses in excess of 32 GiB. Keep this in mind when selecting a
+# non-default instance type.
+ec2-build-farm:
+    build-farm-type: aws-ec2
+    args:
+        # REQUIRED: instance type to use per build
+        instance-type: z1d.2xlarge
+        # REQUIRED: instance market to use per build
+        build-instance-market: ondemand
+        # REQUIRED: if using spot instances, determine the interrupt behavior
+        spot-interruption-behavior: terminate
+        # REQUIRED: if using spot instances, determine the max price
+        spot-max-price: ondemand
+        # REQUIRED: default location of build directory on build host
+        build-dir: /home/centos/firesim-build
+
+# Unmanaged list of build hosts. Assumed that they are pre-setup to run builds.
+local-build-farm:
+    build-farm-type: unmanaged
+    args:
+        # REQUIRED: default location of build directory on build host
+        default-build-dir: /home/centos/firesim-build
```

Comment:
if this is a requirement of all build farms anyway, pull it up a level? e.g.

    build-farm-type: unmanaged
    default-build-dir: ...
    custom-args:
        ...

also, name it consistently between ec2-build-farm and local-build-farm

Reply:
This is not necessarily a requirement for all build farms (I can see super custom build farms not need to have this).